### PR TITLE
Auto Update Site Extension

### PR DESCRIPTION
**Pull request by shurivich**
Add action, which will update automatically site extension on web site for perf testing.
The flow is:

Create nuget package with id depends on run id.
Push to AzDo feed.
Update Site Extension from AzDo feed.

Action will be triggered by `dd-trace-dotnet` nightly build.

### CODE REVIEW

File: .github/workflows/update_aas_extension.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
+name: Update extension on AAS, triggered by dd-trace-dotnet nightly build
+
+on:
+  workflow_dispatch:
+    inputs:
+      sha:
+        description: 'Hash commit of dd-trace-dotnet build'
+        required: true
+  repository_dispatch:
+    types: [dd-trace-dotnet-release]
```
Comment (pierotibou) [inline .github/workflows/update_aas_extension.yml:10]: Change this one to dd-trace-dotnet-nightly for instance

File: dotnet/build-packages-dev.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
+DEVELOPMENT_VERSION="0.2.114-prerelease"
```
Comment (pierotibou) [inline dotnet/build-packages-dev.sh:1]: NIt - if the build part is always replaced automatically and never commited, we could maybe replace is by a `x` and update the regex in the action

File: .github/workflows/update_aas_extension.yml [hunk 3]
```diff
@@ -0,0 +1,92 @@
+name: Update extension on AAS, triggered by dd-trace-dotnet nightly build
```
Comment (pierotibou) [inline .github/workflows/update_aas_extension.yml:1]: NIT - as discussed could be better to have this in the gitlab pipeline. As we would have the dd-trace-dotnet gitlab thee rel env and aas pipelines on nightly builds. But clearly not an issue for now

File: dotnet/build-packages-dev.sh [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+AGENT_DOWNLOAD_URL="http://s3.amazonaws.com/dsd6-staging/windows/agent7/buildpack/agent-binaries-7.35.2-1-x86_64.zip"
+TRACER_DOWNLOAD_URL="https://apmdotnetci.blob.core.windows.net/apm-dotnet-ci-artifacts-master/INSTALL_SHA/windows-tracer-home.zip"
```
Comment (pierotibou) [inline dotnet/build-packages-dev.sh:3]: In the long run we will want to merge this script and the prod one and just pass variables I assume

**Review by pierotibou — DISMISSED** [review]

File: .github/workflows/update_aas_extension.yml (hunk 1 above)
Comment (shurivich) [reply .github/workflows/update_aas_extension.yml]: renamed in 852851287bb3961484816f0401baaf97bdab7546

**Review by pierotibou — APPROVED** [review]

**Pull request merged**